runtimes/native/src/apu.c

```diff
@@ -96,6 +99,9 @@ void w4_apuTone (int frequency, int duration, int volume, int flags) {
     int decay = (duration >> 16) & 0xff;
     int attack = (duration >> 24) & 0xff;
 
+    int sustainVolume = volume & 0xffff;
+    int peakVolume = (volume >> 16) & 0xffff;
```

Comment:
This should probably be set to 100 by default or it will change behavior of current carts

Reply:
Not sure how to implement this TBH. If the peak volume input is 0. how can I tell if it's the user's input or no input?

Reply:
Having it set to 0 would mean that the attack phase is just silent and that the decay phase essentially becomes an attack phase. You don't gain anything from being able to do this.

Reply:
Well, I guess the way to look at it is, it just behaves the same as before but if you want you can set a peak volume as well. From that perspective it makes sense, but from a fresh user's perspective, it's weird that setting the peak to 0 results in full peak volume. This should be reflected in the docs.

Reply:
Yea, just say it will default to 100%. Not really that different from pitch bend not accepting a 0 either.